### DISCUSSION TITLE
Add non-recipes to meal plan

### DISCUSSION THREAD

**Discussion by venom85** [mealie-recipes/mealie#1961] [Feature Request]
Apologies if this has been discussed before (I didn't see it, but I've been blind before...), but it would be very nice to be able to write in entries to a meal plan that are not already recipes. For instance, my dinner tonight will include store-bought dinner rolls. I want to make the meal plan complete, both for my family to know what's coming and so I can make sure I have everything I need on the shopping list ahead of time. I can't add those right now, though, unless I create a recipe for them. That obviously is rather silly for something I buy in the store and then put directly on a plate with no preparation or cooking.

Similarly, it would be nice to be able to indicate that a particular night is an "eat out" night at a restaurant. Right now, I do this by creating "dummy" recipes for restaurants that are in a Restaurants category. It would be nicer to just be able to write in "Portillo's" (for anyone living around Chicago in the United States) on a particular day instead, though.

Love the app, though! This is so much more convenient for the family than a standalone app on just my phone!

**Comment by Ravio1i** [comment]
Hi, I think your feature already exists. When you add a new entry to your meal plan you can click **Note only** and then you don't need to link a recipe, and can just set your title and note.
![image](https://user-images.githubusercontent.com/45916614/212532457-f18ff11e-2db4-4335-ba19-1e4c1632703e.png)
